packages/schematics/angular/migrations/update-8/drop-es6-polyfills.ts

```diff
@@ -73,4 +91,43 @@ export const dropES2015Polyfills = (): Rule => {
 
     tree.commitUpdate(recorder);
   };
-};
+}
+
+/**
+ * Move the import reflect metadata polyfill from the polyfill file to the dev environment. This is
+ * not guaranteed to work, but if it doesn't it will result in no changes made.
+ */
+export function dropES2015Polyfills(): Rule {
+  return (tree) => {
+    // Simple. Take the ast of polyfills (if it exists) and find the import metadata. Remove it.
+    const angularConfigContent = tree.read('angular.json') || tree.read('.angular.json');
+    const rules: Rule[] = [];
+
+    if (!angularConfigContent) {
+      // Is this even an angular project?
+      return;
+    }
+
+    const angularJson = parseJson(angularConfigContent.toString(), JsonParseMode.Loose);
+
+    if (!isJsonObject(angularJson) || !isJsonObject(angularJson.projects)) {
+      // If that field isn't there, no use...
+      return;
+    }
+
+    // For all projects
+    for (const projectName of Object.keys(angularJson.projects)) {
+      const project = angularJson.projects[projectName];
+      if (!isJsonObject(project)) {
+        continue;
+      }
+      if (typeof project.sourceRoot != 'string' || project.projectType !== 'application') {
+        continue;
+      }
+
+      rules.push(dropES2015PolyfillsFromFile(join(normalize(project.sourceRoot), '/polyfills.ts')));
```

Comment:
This should be checking the `polyfills` option.  The file may have been renamed or moved.